README.md

```diff
@@ -83,19 +83,85 @@ oradewrc.json           Workspace configuration
 
 ### DB environment
 
-Only `dbconfig.json` file is required for the workspace activation and successful connection with your database. Multiple DB environments with multi-users per environment are supported. A minimal example with `DEV` environment and a single schema user follows:
+Only `dbconfig.json` file is required for the workspace activation and successful connection with your database. Multiple DB environments with multi-users per environment are supported. `DEV` environment will always be used for importing existing sources from DB, other environment names are not allowed. A minimal example with `DEV` environment and a single schema user follows:
 
 ```json
 {
-  "DEV": {
-    "connectString": "localhost/orclpdb",
-    "users": [{ "user": "hr", "password": "welcome" }]
-  }
+   "DEV":{
+      "connectString":"localhost/orclpdb",
+      "users":[
+         {
+            "user":"hr",
+            "password":"welcome"
+         }
+      ]
+   }
 }
 ```
 
+For credentials stored in oracle wallets, or auto-login oracle wallets:
```

Comment:
I was thinking we should restructure readme as it is already too long and difficult too read for new users... Maybe we should consider wiki, what do you think?

Reply:
Yes, perfect,  better in the wiki as it is not directly related to oradew.